src/Main.java

```diff
@@ -2677,10 +2773,17 @@ else if (region == 0x1a) {
                     return difficulty >= Difficulty.S_HARD || inventory.contains(Items.JUMP_BOOTS);
                 }
                 else {
-                    // Secret Attic will never be required on Easy. Hard may require you to nudge enemies up to the top
-                    // without a glove
-                    return difficulty > Difficulty.EASY && inventory.contains(Items.JUMP_BOOTS)
-                            && (difficulty >= Difficulty.HARD || canLift(inventory));
+				/*
```

Comment:
I think the logic that was there before correctly handles this case, and a little bit more efficiently - is there a reason for this refactor?

Reply:
Don't know why it looked like that on my side. May have been because my code was still based off my original forking of Version v0.11.5. I have fixed this.